--- malloc3.stderr.exp	2019-05-21 15:39:40.714369949 +0300
+++ malloc3.stderr.res	2019-05-21 16:38:47.424837633 +0300
@@ -1,8 +1,22 @@
-Argument 'size' of function malloc has a fishy (possibly negative) value: -1
-   at 0x........: malloc (vg_replace_malloc.c:...)
-   by 0x........: main (malloc3.c:15)
-
-Argument 'size' of function calloc has a fishy (possibly negative) value: -1
-   at 0x........: calloc (vg_replace_malloc.c:...)
-   by 0x........: main (malloc3.c:23)
-
+==7802== Memcheck, a memory error detector
+==7802== Copyright (C) 2002-2017, and GNU GPL'd, by Julian Seward et al.
+==7802== Using Valgrind-3.16.0.GIT and LibVEX; rerun with -h for copyright info
+==7802== Command: ./malloc3
+==7802== 
+==7802== Argument 'size' of function malloc has a fishy (possibly negative) value: -1
+==7802==    at 0x4C2BF3D: malloc (vg_replace_malloc.c:309)
+==7802==    by 0x1087D5: main (in /home/dyu/git/Scripts/vg_regtests/small_tests/memcheck/malloc3)
+==7802== 
+==7802== Argument 'size' of function calloc has a fishy (possibly negative) value: -1
+==7802==    at 0x4C2E18C: calloc (vg_replace_malloc.c:762)
+==7802==    by 0x108845: main (in /home/dyu/git/Scripts/vg_regtests/small_tests/memcheck/malloc3)
+==7802== 
+==7802== 
+==7802== HEAP SUMMARY:
+==7802==     in use at exit: 0 bytes in 0 blocks
+==7802==   total heap usage: 3 allocs, 3 frees, 4,096 bytes allocated
+==7802== 
+==7802== All heap blocks were freed -- no leaks are possible
+==7802== 
+==7802== For lists of detected and suppressed errors, rerun with: -s
+==7802== ERROR SUMMARY: 2 errors from 2 contexts (suppressed: 0 from 0)
